### PR TITLE
fix cluster/images/etcd/migrate staticcheck

### PR DESCRIPTION
**Pull request by hwdef**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

fix staticcheck 

```shell
# edward @ edward-pc in ~/code/go/kubernetes/cluster on git:master o [10:27:43] 
$ staticcheck ./...
images/etcd/migrate/versions.go:129:6: func MustParseEtcdVersionPair is unused (U1000)
images/etcd/migrate/versions.go:191:6: func MustParseSupportedVersions is unused (U1000)
```

**Which issue(s) this PR fixes**:

Part of #92402

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: cluster/images/etcd/migrate/versions.go [hunk 1]
```diff
@@ -124,16 +124,6 @@ func ParseEtcdVersionPair(s string) (*EtcdVersionPair, error) {
 	return &EtcdVersionPair{version, storageVersion}, nil
 }
 
-// MustParseEtcdVersionPair parses a "<version>/<storage-version>" string to an EtcdVersionPair
-// or panics if the parse fails.
-func MustParseEtcdVersionPair(s string) *EtcdVersionPair {
```
Comment (BenTheElder) [inline cluster/images/etcd/migrate/versions.go:129]: we should check cs.k8s.io to see if anyone else imports this

**Comment by hwdef** [comment]
@BenTheElder 

I found these function are used. So I will close this PR.

**Comment by BenTheElder** [comment]
wait, where are they used?
I see:
- vendor (ignore)
- a fork of cluster/ in cloud-provider-gcp
- this repo
https://cs.k8s.io/?q=MustParseEtcdVersionPair&i=nope&files=&repos=

**Comment by hwdef** [comment]
@BenTheElder 
Yes, you are right.
I found `cluster/images/etcd/migrate/integration_test.go` use these two functions. Should I delete these functions?

**Pull request closed without merging**